.github/CODEOWNERS

```diff
@@ -0,0 +1,34 @@
+###################################
+##### Global Protection Rule ######
+###################################
+# NOTE: This rule is overriden by the more specific rules below. This is the catch-all rule for all files not covered by the more specific rules below
+*                                              @hashgraph/developer-advocates
```

Comment:
getting errors about teams not having access to this repo

Reply:
Added teams